src/index.html

```diff
@@ -7,6 +7,325 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a class="header__logo" href="#home">
+        <img 
+        class="logo" 
+        src="images/logo.png" 
+        alt="logo"
+        >
+      </a>
+      <nav class="nav">
```

Comment:
Don't forget to add empty lines between multiline sibling blocks of HTML
```suggestion
      </a>
      
      <nav class="nav">
```